start/sw/web-app/django.md

```diff
@@ -10,130 +10,103 @@ Django is a high-level Python Web framework that encourages rapid development an
 Django is one of the most popular Python Web Frameworks. It has very useful web page. Visit it by clicking on [this link](https://www.djangoproject.com/). 
 Because you will need to work with Python packages, write this command to your command line.
 
-For Python:
-
-```
-sudo dnf install python-pip
-```
-
-or Python 3:
-
 ```
-sudo dnf install python3-pip
+$ sudo dnf install python3-pip
 ```
 
-# Get started
+## Get started
 
-## How to install virtualenv
+### How to install virtualenv
 
 It's recommended to keep your project inside virtual environment. But you can skip this step if you want. 
 Enter this command to your command line for installation:
 
 ```
-sudo dnf install python-virtualenv
-```
-
-or installation for Python3:
-
-```
-sudo dnf install python3-virtualenv
+$ sudo dnf install python3-virtualenv
 ```
 
 
 Get inside your working directory or create new where you want to store your virtual environment:
 
 ```
-mkdir working-dir
-cd working-dir
+$ mkdir working-dir
+$ cd working-dir
 ```
 
 Now we will create virtual environment called venv inside this directory using following command:
 
 ```
-virtualenv venv
+$ virtualenv venv
 ```
 
 When it's done, we will activate our virtualenv. Now we will use following commands:
 
 ```
-cd venv
-source bin/activate
+$ cd venv
+$ source bin/activate
 ```
 
 When it's activated we will see indicator of same name as our virtualenv. 
 Now we will check if we really have empty environment:
 
 ```
-pip freeze
+$ pip freeze
 ```
 
 It will return nothing.
 
-## How to install Django
+### How to install Django
 
 Now if you installed `virtualenv` do not leave it, we will install Django now. If you installed `virtualenv` use this command 
 inside your working directory with activated `virtualenv`:
 
 ```
-pip install django
+$ pip3 install django
 ```
 
 If you are following a book or a tutorial, you might want to specify the same exact version like this
 
 ```
-pip install django==1.10
-```
-
-For Python 3:
-
-```
-pip3 install django==1.10
+$ pip3 install django==1.10
 ```
 
 
 Of course you can use the version that is prepared by Fedora community by typing:
 
-For Python 2:
-
 ```
-sudo dnf install python-django
-```
-
-or Python 3:
-
-```
-sudo dnf install python3-django
+$ sudo dnf install python3-django
```

Comment:
Could we prefer this instead of `pip`. What do we recommend user to do? Are there some (dis)advantages? I think this should be included.

_[ non-blocking ]_

Reply:
Ok, so maybe state that, instead of [L74](https://github.com/developer-portal/content/pull/242/files/f8fd7eb496b9aabf8a54fa6ed25f2e141ce612dc#diff-a0096ededfb8e6fc0c157dd79efbf8f8R74)?